test-resources/cljs/run-phantom.js

```diff
@@ -0,0 +1,52 @@
+var system = require('system');
```

Comment:
I can have a look at 'doo' on Monday; testing in Clojurescript is not as straightforward as Clojure. The approach that I used runs a shell process with Phantom js to execute the tests in Javascript.

Doo seems to package up some aspects.